cmd/influx/replication.go

```diff
@@ -22,13 +23,71 @@ func newReplicationCmd() cli.Command {
 }
 
 func newReplicationCreateCmd() cli.Command {
+	var params replication.CreateParams
 	return cli.Command{
 		Name:   "create",
 		Usage:  "Create a new replication stream",
 		Before: middleware.WithBeforeFns(withCli(), withApi(true), middleware.NoArgs),
-		Flags:  commonFlags(),
-		Action: func(ctx *cli.Context) {
-			fmt.Println("replication create command was called")
+		Flags: append(
+			commonFlags(),
+			&cli.StringFlag{
+				Name:        "name, n",
+				Usage:       "Name for new replication stream",
+				Required:    true,
+				Destination: &params.Name,
+			},
+			&cli.StringFlag{
+				Name:        "description, d",
+				Usage:       "Description for new replication stream",
+				Destination: &params.Description,
+			},
+			&cli.StringFlag{
+				Name:        "org-id",
+				Usage:       "The ID of the local organization",
+				EnvVar:      "INFLUX_ORG_ID",
+				Destination: &params.OrgID,
+			},
+			&cli.StringFlag{
+				Name:        "org, o",
+				Usage:       "The name of the local organization",
+				EnvVar:      "INFLUX_ORG",
+				Destination: &params.OrgName,
+			},
+			&cli.StringFlag{
+				Name:        "remote-id",
+				Usage:       "Remote connection ID new replication stream will be registered with",
+				Required:    true,
+				Destination: &params.RemoteID,
+			},
+			&cli.StringFlag{
+				Name:        "local-bucket",
+				Usage:       "Local bucket ID for new replication stream",
+				Required:    true,
+				Destination: &params.LocalBucketID,
+			},
+			&cli.StringFlag{
+				Name:        "remote-bucket",
+				Usage:       "Remote bucket ID for new replication stream",
+				Required:    true,
+				Destination: &params.RemoteBucketID,
+			},
+			&cli.Int64Flag{
+				Name:        "max-queue",
+				Usage:       "Max queue size in bytes",
+				Value:       67108860,
```

Comment:
Is this value code-gen'd out of the API spec anywhere, for us to reuse here? If not could you add a comment pointing to the part of the spec it's copied from?